department-of-veterans-affairs/caseflow
# HearingDay Data Model Implementation

Resolves #6741 

### Description
Implement the HearingDay model and the hearing_days Postgres table. The model needs to be compatible with the HearingDay behavior prior to this issue where data is stored into VACOLS. An implementation date is defined as a constant to control the move from VACOLS to Caseflow data storage.

### Acceptance Criteria
- [ ] Defined tests pass

### Testing Plan
1. Build an RO Schedule
2. Add judge assignments to the schedule
3. Check both FACOLS and Caseflow databases depending on schedule dates.

Do the above for both dates before April 1, 2019 and after, as well as combination. All work.
